frontend/src/features/admin-form/settings/components/WorkflowSettingsSection/WorkflowSettingsSection.tsx

```diff
@@ -0,0 +1,29 @@
+import { Text } from '@chakra-ui/react'
+
+import { FormResponseMode } from '~shared/types'
+
+import { GUIDE_TWILIO } from '~constants/links'
+import Link from '~components/Link'
+
+import { useAdminFormSettings } from '../../queries'
+
+import { WorkflowDetailsInput } from './WorkflowDetailsInput'
+
+export const WorkflowSettingsSection = (): JSX.Element => {
+  const { data: settings, isLoading } = useAdminFormSettings()
```

Comment:
reminder on unused loading props

Reply:
I've deleted this component, realised it's unused! [245d990](https://github.com/opengovsg/FormSG/pull/6968/commits/245d99044c159a86406da30f9c6fcf11dfc11230)